src/clients/c++/request.h

```diff
@@ -329,6 +329,13 @@ class InferContext {
     /// \param input The vector holding tensor string values.
     /// \return Error object indicating success or failure.
     virtual Error SetFromString(const std::vector<std::string>& input) = 0;
+
+    // Set the shard memory object for the batch tensor input
+    /// \param shm_key The shared memory location holding the input tensors.
```

Comment:
If "shared memory location" the terminology used in the API documentation for shm_open and mmap? We should follow whatever naming convention is used there. Also it is not the location "holding the input tensors". It is "holding tensor values for this input".

Reply:
"Offset into the shared memory <whatever> to the start of the input tensor values"